docker/deploy.bat

```diff
@@ -0,0 +1,2 @@
+docker-compose -p our_dspace -f deploy.yml up -d
```

Comment:
I would rather create "our" directory with scripts e.g., we use
`build-scripts/build` `build-scripts/test` etc.

Reply:
used build-scripts